src/DataGrid.tsx

```diff
@@ -284,7 +289,8 @@ function DataGrid<R, SR, K extends Key>(
   /**
    * computed values
    */
-  const [gridRef, gridWidth, gridHeight, isWidthInitialized] = useGridDimensions();
+  const [gridRef, gridWidth, gridHeight] = useGridDimensions();
+  const [prevGridWidth, setPrevGridWidth] = useState(gridWidth);
```

Comment:
Should this be moved with the other states?

Reply:
Need to set the initial state using `gridWidth`

Reply:
The initial `gridWidth` is `1` anyway 🤔